src/Workspaces/Core/Portable/Utilities/Documentation/XmlDocumentationProvider.cs

```diff
@@ -40,7 +47,7 @@ protected override string GetDocumentationForSymbol(string documentationMemberID
                         {
                             if (e.Attribute("name") != null)
                             {
-                                _docComments[e.Attribute("name").Value] = e.Value;
+                                _docComments[e.Attribute("name").Value] = string.Concat(e.Nodes());
```

Comment:
It has the base type of DocumentationProvider, not XmlDocumentationProvider. (I agree the names are terrible.)